translations/ar/LC_MESSAGES/messages.po

```diff
@@ -83,9 +78,8 @@ msgstr "يبدو أنك نسيت استخدام أمر برمجي في السط
 msgid "Missing Inner Command"
 msgstr "يبدو أنك نسيت استخدام أمر برمجي مع الجملة {command} التي استعملتها في السطر رقم {line_number}."
 
-#, fuzzy
 msgid "Missing Variable"
-msgstr "It looks like your {command} is missing a variable at the start of the line."
```

Comment:
Oh this is actually something I'm a bit unfamiliar with. From what I understood these were supposed to be empty and it'll automatically fill in its English counterpart? I will change this however to be as you mentioned!

Reply:
Yes and no (so I understand your confusion!!). 

In the case of a newly added string (like when you added this in #4974) indeed the English string is copied over. But now that the string is added, it becomes a "normal string" and then when it is changed, it will not be copied over. Because maybe you would be making a small change like a typo only in one language. 

In short, now that it is there, we should no longer remove it!